### DISCUSSION TITLE
measuring 11V on 1X probe broke my DSO-500K!

### DISCUSSION THREAD

**Discussion by emadalahdab** [fhdm-dev/scoppy#132] [General]
Hallo,
Last week I bought the DSO-500K and it's my first oscilloscope, i was doing some testing on 11V DC audio waveform and i forgot to put the probe on x10 and now both channels doesn't measure voltages over 5.49 V.
Any idea if it possible to fix the problem and where to start?

I was thinking maybe making an analog front end might fix the problem, but still i would like to try to fix it as a learning experience 

Thanks in advance

**Comment by fhdm-dev** [comment]
Hi,

The DSO-500K hardware should easily be able to cope with 11V on the inputs even with the probe set to 1X so I suspect its likely a software/firmware issue.

I first thing I think you should do is check the voltage ranges in the app in case they have somehow gone awry. Tap the channel badge at the bottom of the screen and then tap 'Select input voltage range'. This will show the min and max voltages (that will be displayed in the app) for the selected range and probe attenuation (ie. the probe attenuation that is configured in the app). For voltage range 0, the range  should be -5.872V - 5.917V assuming the probe attenuation in the app is set to 1X. The other ranges should be -2.152V -2.497V, -1.121V - 0.949V and -0.404V to 0.595V.

Also I should mention that the app sometimes doesn't keep the probe attenuation setting after it has been closed or put in the background. So for example, if you have your actual probe set to 10X and the setting in the app has gone back to 1X then obviously it's going to show the wrong voltages.

To start troubleshooting the hardware itself you'll first want to look at the schematic which can be found [here](https://oshwlab.com/fruitloop57/fscope-250k5-v2_copy_copy_copy_copy).

To start testing, connect your 10X probe to a voltage of say 12V, ensure the scope is in RUN mode (ie. not STOPPED) and then check the voltages at the input and the output of each of the op-amps. Given that you want this to be a learning experience I won't tell you what the voltages should be though I'm happy to do that if you want :)

Be aware that the selected voltage range in the app affects voltages you would can expect to see on the board. Manually selecting a voltage range in the app can make the troubleshooting process easier.

I hope that's enough to get you started. Please let me know if you need more information.

**Reply by emadalahdab** [reply]
Hallo,
thank you for helping, i looked up "Select input voltage range" and All works just right!
For the trying to check the voltage on the op-amps , i wasn't able to check it on the U1 when the U2 gives around a 0,95V on the input side and around 2V on the output pin all this while connecting the x10 Probe to 12V power supply. I need to dig into op-amps Funktion more after finishing exams

Thanks again for helping
